addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -340,50 +359,45 @@ def _plan_filter_activity_templates_to_schedule(self):
+                if self.plan_date:
+                    next_line.summary_date_deadline = activity_type.triggered_next_type_id._get_date_deadline() + (self.plan_date - fields.Date.context_today(self))
+                if activity_type.triggered_next_type_id.default_user_id:
+                    next_line.summary_user_id = activity_type.triggered_next_type_id.default_user_id
             # Suggested next activities
             elif activity_type.suggested_next_type_ids:
-                suggested_activities = []
-                for suggested_activity in activity_type.suggested_next_type_ids:
-                    suggested_delay_unit = dict(suggested_activity._fields['delay_unit']._description_selection(self.env))[suggested_activity.delay_unit]
-                    suggested_delay_from = dict(suggested_activity._fields['delay_from']._description_selection(self.env))[suggested_activity.delay_from]
-                    suggested_activities.append(
-                        _("%(activity_name)s %(delay_count)s %(delay_unit)s %(delay_from)s",
-                        activity_name=suggested_activity.name,
-                        delay_count=suggested_activity.delay_count,
-                        delay_unit=suggested_delay_unit,
-                        delay_from=suggested_delay_from
-                        )
-                    )
-                next_activities.append(_(" or ").join(suggested_activities))
-            # Add next activities as nested list for each activity type
-            if next_activities:
-                nested_summary_line = Markup('<ul>%s</ul>') % Markup().join(
-                    Markup('<li>%s</li>') % activity for activity in next_activities
-                )
-                summary_line += nested_summary_line
-            summaries.append(Markup('<li>%s</li>') % summary_line)
-        return Markup('<ul>%s</ul>') % Markup().join(summaries) if summaries else ''
+                for suggested in activity_type.suggested_next_type_ids:
+                    next_line = summaries.new({'schedule_id': self, 'depth': 1})
+                    summaries += next_line
+                    next_line.line = suggested.summary or suggested.activity_type_id.name
+                    if self.plan_date:
+                        next_line.summary_date_deadline = suggested._get_date_deadline() + (self.plan_date - fields.Date.context_today(self))
+                    if suggested.default_user_id:
+                        next_line.summary_user_id = suggested.default_user_id
+        return summaries
+
+
+class MailActivityScheduleSummary(models.TransientModel):
+    _name = 'mail.activity.schedule.summary'
+    _description = 'Mail Activity Schedule Summary Line'
+    schedule_id = fields.Many2one('mail.activity.schedule')
+
+    depth = fields.Integer(default=0)
+    line = fields.Char()
+    summary_date_deadline = fields.Date()
+    summary_user_id = fields.Many2one('res.users')
```

Comment:
Just call it user_id, summary is namespace is not necessary in this model's context